### DISCUSSION TITLE
Links to Google slides/sheets

### DISCUSSION THREAD

**Discussion by glazet** [glazet/QuadEVCharger#16] [General]
Here is where the links for the Smart Quad EV Charger google documents

**Comment by glazet** [comment]
[PowerPoint](https://docs.google.com/presentation/d/1tK7BEZV_6pD6SzRpXh4eQvX8X0hkCamg/edit?usp=sharing&ouid=105807759067601935006&rtpof=true&sd=true)
Updated to Slide 11

[Materials Sheet](
https://docs.google.com/spreadsheets/d/10eQG6eU977sr-Vo4AyLgy35bYP38K_eaPXTxfUFYXaI/edit?usp=sharing)

[Smart Quad EV Charger Report](
https://docs.google.com/document/d/1f07KjNocCNgrVMPTOWqJHpus8bWcwUXt/edit?usp=sharing&ouid=101153106971570841716&rtpof=true&sd=true)
Slowly updating

**Comment by djangodemetri** [comment]
[Minimum Viable Product Schedule](https://docs.google.com/spreadsheets/d/17dWSB3O24H2Vjkp8bGaF0v0rzq5ACB4X7yRKv1XJ_DY/edit?usp=sharing)
